src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ClaimantTellSomethingElseService.java

```diff
@@ -121,28 +172,207 @@ public String buildApplicationCompleteResponse(CaseData caseData) {
+    private Map<String, String> prepareEmailContent(CaseDetails caseDetails) {
+        CaseData caseData = caseDetails.getCaseData();
+
+        Map<String, String> content = new ConcurrentHashMap<>();
+        content.put(CASE_NUMBER, caseData.getEthosCaseReference());
+        content.put(CLAIMANT, caseData.getClaimant());
+        content.put(RESPONDENT_NAMES, getRespondentNames(caseData));
+        content.put(HEARING_DATE, getNearestHearingToReferral(caseData, NOT_SET));
+        content.put(SHORT_TEXT, caseData.getClaimantTseSelectApplication());
+        content.put(EXUI_CASE_DETAILS_LINK, emailService.getExuiCaseLink(caseDetails.getCaseId()));
+
+        return content;
+    }
+
+    public void sendRespondentsEmail(CaseDetails caseDetails) {
+        CaseData caseData = caseDetails.getCaseData();
+        String applicationType = caseData.getClaimantTseSelectApplication();
+
+        if (CLAIMANT_TSE_ORDER_A_WITNESS_TO_ATTEND.equals(applicationType)
+            || NO.equals(caseData.getClaimantTseRule92())
+            || NO.equals(caseData.getClaimantTseRespNotAvailable())) {
+            return;
+        }
+
+        List<String> respondentEmailAddressList = getRespondentEmailAddressList(caseData);
+        boolean isWelsh = featureToggleService.isWelshEnabled()
+                && WELSH_LANGUAGE.equals(caseData.getClaimantHearingPreference().getContactLanguage());
+
+        try {
+            byte[] bytes = tornadoService.generateEventDocumentBytes(caseData, "", TSE_FILE_NAME);
+            String emailTemplate = getRespondentEmailTemplate(isWelsh, applicationType);
+
+            respondentEmailAddressList.forEach(respondentEmail ->
+                    sendEmailToRespondent(
+                            emailTemplate,
+                            respondentEmail,
+                            caseDetails,
+                            bytes,
+                            isWelsh));
+        } catch (Exception e) {
+            throw new DocumentManagementException(String.format(DOCGEN_ERROR, caseData.getEthosCaseReference()), e);
+        }
+    }
+
+    private void sendEmailToRespondent(String emailTemplate, String respondentEmail, CaseDetails caseDetails,
+                                       byte[] bytes, boolean isWelsh) {
+        CaseData caseData = caseDetails.getCaseData();
+        try {
+            emailService.sendEmail(
+                    emailTemplate,
+                    respondentEmail,
+                    prepareRespondentEmailContent(caseDetails, bytes, isWelsh)
+            );
+        } catch (NotificationClientException e) {
+            throw new DocumentManagementException(String.format(DOCGEN_ERROR, caseData.getEthosCaseReference()), e);
         }
+    }
 
-        AtomicInteger applicationNumber = new AtomicInteger(1);
+    private List<String> getRespondentEmailAddressList(CaseData caseData) {
+        List<String> respEmailAddresses = new ArrayList<>();
+        caseData.getRespondentCollection()
```

Comment:
I think a change might be needed here to send the email to the respondent rep only if they are on MyHMCTS.

Reply:
Otherwise we could potentially be sending emails to users that do not have access to the portal